### ISSUE TITLE
Implementation missing for the class RichCodeEditingController.

### ISSUE THREAD

**Issue by hipoojan** [psovit/rich_code_editor#19]
```
Error: The non-abstract class 'RichCodeEditingController' is missing implementations for these members:
 - TextEditingController.isSelectionWithinTextBounds
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class RichCodeEditingController extends ValueNotifier<TextEditingValue> implements TextEditingController {
      ^^^^^^^^^^^^^^^^^^^^^^^^^
/D:/flutter/packages/flutter/lib/src/widgets/editable_text.dart:225:8: Context: 'TextEditingController.isSelectionWithinTextBounds' is defined `here.`
  bool isSelectionWithinTextBounds(TextSelection selection) {
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^``
```

**Comment by swapanil** [comment]
Same issue i am facing too
<img width="571" alt="Screenshot 2020-08-05 at 9 56 06 PM" src="https://user-images.githubusercontent.com/42803028/89438494-823ea100-d766-11ea-9628-65a295895ff0.png">
<img width="575" alt="Screenshot 2020-08-05 at 9 56 18 PM" src="https://user-images.githubusercontent.com/42803028/89438514-8a96dc00-d766-11ea-8aca-8324c386a6b9.png">

**Comment by PegasisForever** [comment]
If you are using flutter 1.20, there's a temporary solution:

Change your dependency to:
```
  rich_code_editor:
    git:
      url: https://github.com/redsolver/rich_code_editor.git
```

**Comment by psovit** [comment]
This issue is now resolved with the latest upgrade to Flutter 1.22.5. Please use `rich_code_editor 1.0.5`. Thanks.
